app/src/main/java/org/kinecosystem/kinit/analytics/Analytics.java

```diff
@@ -28,7 +28,10 @@
     public static final String TRANSACTION_TYPE_SPEND = "spend";
     public static final String P2P_EXCEED_MIN_MAX = "Exceed max/min Kin";
     public static final String P2P_FRIEND_HAS_NO_ADDRESS = "Friend not exists";
+    public static final String P2P_SEND_KIN_TO_SELF = "Send Kin to itself";
```

Comment:
is this error type defined on amplitude? and does @natanrolnik  use the same ?

Reply:
@berryve these are defined in AirTable, we need to add it there.